### DISCUSSION TITLE
Conduit issue

### DISCUSSION THREAD

**Discussion by lidaamber** [ProjectOpenSea/seaport#623] [Q&A] [answered]
Hi, I created a conduit key based on the description [here](https://github.com/ProjectOpenSea/seaport/discussions/544), but when this key is used in the signed order offchain and in order parameters included into transaction, the Seaport contract reverts. I'm trying to fulfill a basic order, fixed-price sale of NFT for ETH. The approval for NFT was given to the conduit in this case, so it's likely not the approval issue
It doesn't if the conduit key is set to 0 bytes in both offchain and onchain parameters and approvals are given to the Seaport contract directly
In ConduitController linked to Seaport contract the conduit that is used has exists = true, so it likely is valid

After creating a conduit like described [here](https://github.com/ProjectOpenSea/seaport/discussions/544), are there any steps required to do afterwards to make it work with Seaport? What might be the potential issue with conduit key? Thanks

**Comment by 0age** [comment, marked as answer]
The owner of the conduit needs to add Seaport as a _channel_ on the conduit before it can be utilized by Seaport!
